### PR TITLE
154167713 käyttöohjeen linkki headeriin 

### PR DESCRIPTION
**Pull request by K1ll3rF0x**

### CODE REVIEW

File: nap/docker/ckan/ckan-plugins/ckanext-napote_theme/ckanext/napote_theme/templates/header.html [hunk 1]
```diff
@@ -70,6 +69,13 @@
 
                         {% block header_account_container_content %}
                             {% if c.userobj %}
+                                <ul class="nav nav-pills">
+                                    <li>
+                                        <a href="https://s3.eu-central-1.amazonaws.com/ote-assets/nap-ohje.pdf" target="_blank" title="NAP-palvelun käyttöohje">
```
Comment (rckopteri) [inline nap/docker/ckan/ckan-plugins/ckanext-napote_theme/ckanext/napote_theme/templates/header.html]: Translate title

File: nap/docker/ckan/ckan-plugins/ckanext-napote_theme/ckanext/napote_theme/templates/header.html [hunk 2]
```diff
@@ -145,6 +146,11 @@
                                 </ul>
                             {% else %}
                                 <ul class="nav nav-pills">
+                                    <li>
+                                        <a href="https://s3.eu-central-1.amazonaws.com/ote-assets/nap-ohje.pdf" target="_blank" title="NAP-palvelun käyttöohje">
```
Comment (rckopteri) [inline nap/docker/ckan/ckan-plugins/ckanext-napote_theme/ckanext/napote_theme/templates/header.html]: here as well

File: nap/docker/ckan/ckan-plugins/ckanext-napote_theme/ckanext/napote_theme/templates/header.html (hunk 1 above)
Comment (K1ll3rF0x) [reply nap/docker/ckan/ckan-plugins/ckanext-napote_theme/ckanext/napote_theme/templates/header.html]: I Copy-pasted the old implementation. I'll add the missing translation key.

File: nap/docker/ckan/ckan-plugins/ckanext-napote_theme/ckanext/napote_theme/templates/header.html (hunk 2 above)
Comment (K1ll3rF0x) [reply nap/docker/ckan/ckan-plugins/ckanext-napote_theme/ckanext/napote_theme/templates/header.html]: I Copy-pasted the old implementation. I'll add the missing translation key.

**Pull request merged**